### PR TITLE
Update nodes list from iotanode.host

### PR DESCRIPTION
**Pull request by avihaiandroid**
Update nodes list from iotanode.host (added only sync and active)

### CODE REVIEW

File: src/transactionSpammer.js [hunk 1]
```diff
@@ -26,18 +26,68 @@ window.iotaTransactionSpammer = (function(){
     // from 'https://iotasupport.com/providers.json' + requested additions - unreliable nodes
     // message me on Slack (Peter Ryszkiewicz) or make an issue here is you want your node added: https://github.com/pRizz/iota.transactionSpammer.js/issues
     const httpProviders = [
-        "http://service.iotasupport.com:14265",
+		"http://iota-community.org:14265",
```
Comment (pRizz) [inline src/transactionSpammer.js]: Can you fix the whitespace on this and following nodes?

File: src/transactionSpammer.js [hunk 2]
```diff
@@ -26,18 +26,68 @@ window.iotaTransactionSpammer = (function(){
...
+        "http://5.189.128.164:14265",
+        "http://heimelaga.vodka:14265",
+        "http://iotausa.mooo.com:14265",
+        "http://iota.band:14265",
         "http://iotanode.prizziota.com:80", // author's node :)
     ]
 
     const httpsProviders = [
+        "https://node.iota-community.org:443",
+		"https://iotanode.us:443",
```
Comment (pRizz) [inline src/transactionSpammer.js]: Fix whitespace

**Review by avihaiandroid — COMMENTED** [review]
Fix spaces

**Comment by avihaiandroid** [comment]
Fixed

**Review by pRizz — APPROVED** [review]

**Pull request merged**